tests/performance/test_api/test_api.py

```diff
@@ -23,7 +38,9 @@ def test_api_endpoints(test_case, test_configuration, set_api_test_environment,
     """Make an API request for each `test_case`. `test_configuration` fixture is only used to add metadata to the
     HTML report."""
     # Apply xfails
-    test_case['endpoint'] in xfailed_items and pytest.xfail(xfailed_items[test_case['endpoint']])
+    test_case['endpoint'] in xfailed_items.keys() and \
+    test_case['method'] == xfailed_items[test_case['endpoint']]['method'] and \
+    pytest.xfail(xfailed_items[test_case['endpoint']]['message'])
```

Comment:
```
20:26: E127 continuation line over-indented for visual indent
23:26: E127 continuation line over-indented for visual indent
26:26: E127 continuation line over-indented for visual indent
29:26: E127 continuation line over-indented for visual indent
42:5: E122 continuation line missing indentation or outdented
43:5: E122 continuation line missing indentation or outdented
```

Reply:
@AdriiiPRodri Can you have a look at this?